packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -36,22 +36,39 @@
                     :class="$style['c-checkout-form']"
                     @submit.prevent="onFormSubmit"
                 >
-                    <guest-block v-if="!isLoggedIn" />
+                    <section
+                        id="error-summary-container"
+                        :class="$style['is-visuallyHidden']"
```

Comment:
Not 100% sure on this, but as these styles come from fozzie and are shared, should this be: (/cc @dandel10n )

```suggestion
                        :class="is-visuallyHidden"
```